src/connect4/core.rs

```diff
@@ -445,7 +470,11 @@ pub struct GameState {
     gameLoaded: GameLoaded,
     /// connect4 board
     pub board: Board,
+    team_colors: Vec<MyColor>,
     pub turnIndicator: TurnIndicator,
+    pub highlighted_column: i32,
+    mouse_disabled: bool
+
```

Comment:
Added some fields to track what colors to use for each team, which column the mouse is currently highlighting, and should the mouse be disabled (this was intended for pausing, but not really implemented yet)